com.github.Bleuzen.FFaudioConverter.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id": "com.github.Bleuzen.FFaudioConverter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "ffaudioconverter",
+    "rename-desktop-file": "ffaudioconverter.desktop",
+    "rename-appdata-file": "FFaudioConverter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "libmp3lame",
+            "config-opts": ["--enable-shared",
+                            "--enable-nasm",
+                            "--bindir=/app/bin"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/lame/lame-3.100.tar.gz",
+                    "sha256": "ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                "--enable-gpl",
+                "--disable-static",
+                "--enable-shared",
+                "--disable-doc",
+                "--disable-ffplay",
+                "--disable-ffprobe",
+                "--enable-libmp3lame",
+                "--enable-libvorbis",
+                "--enable-libopus"
+            ],
+            "cleanup": [
+                "/share/ffmpeg/examples"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://ffmpeg.org/releases/ffmpeg-4.1.tar.xz",
+                "sha256": "a38ec4d026efb58506a99ad5cd23d5a9793b4bf415f2c4c2e9c1bb444acd1994"
+                }
+            ]
+        },
+        {
+            "name": "FFaudioConverter",
+            "buildsystem": "simple",
```

Comment:
Use `qmake` build system.

Reply:
I prefer a script, because if I want to change something in the future I will only have to change it once, in the script, and flatpak version will also have this build change.
Also I can extra commands like sed to change defaults in the flatpak version. So I like the simple buildsystem more.
Can I keep it?

Reply:
It really doesn't make any sense at all. All sane software builds with two commands: `qmake ...`, `make install`. If anything else is required you did something wrong.

Reply:
Also `sed` is not how you build software either. You have a build system that detects system state and you have a compiler that takes variables, they should be used.

Reply:
How to detect Flatpak during build.. does Flatpak set any variable, that I could check in code with `#ifdef` or will I have to set my own build flag?

Reply:
You can check in the buildsystem if the file `/.flatpak-info` exists.

It should be pretty rare to actually care that you are in flatpak though. I see no reason this application should need to know that. I'm not super familiar with QMake but in CMake for example you would `find_program(ffmpeg, ...)`.

Reply:
Well I'm also pretty new to QMake (this is my first Qt and C++ application), but after some time asking my search engine, I did not find something like cmakes find_program in QMake.
So I guess I will have to check for that `/.flatpak-info`.

Reply:
Hmm, still pretty broken (plenty of systems can use something other than `/usr` but for the scope of this package I guess its fine.